.gitignore

```
@@ -12,8 +12,12 @@ venv/
 *.json
 *.csv
 *.zip
-*.xls
-*.xlsx
+/*.xls
+/*.xlsx
```

Faltou voltar esses aqui ao que estava antes. 😅 

```suggestion
*.xls
*.xlsx
```